fenix/app/src/main/res/values/strings.xml

```diff
@@ -478,11 +478,15 @@
     <!-- Preference for accessibility -->
     <string name="preferences_accessibility">Accessibility</string>
     <!-- Preference to override the Firefox Account server -->
-    <string name="preferences_override_fxa_server">Custom Firefox Account server</string>
+    <string name="preferences_override_fxa_server" moz:RemovedIn="120" tools:ignore="UnusedResources">Custom Firefox Account server</string>
+    <!-- Preference to override the Mozilla Account server -->
+    <string name="preferences_override_account_server">Custom Mozilla Account server</string>
     <!-- Preference to override the Sync token server -->
     <string name="preferences_override_sync_tokenserver">Custom Sync server</string>
     <!-- Toast shown after updating the FxA/Sync server override preferences -->
-    <string name="toast_override_fxa_sync_server_done">Firefox Account/Sync server modified. Quitting the application to apply changes…</string>
+    <string name="toast_override_fxa_sync_server_done" moz:RemovedIn="120" tools:ignore="UnusedResources">Firefox Account/Sync server modified. Quitting the application to apply changes…</string>
+    <!-- Toast shown after updating the Mozilla Account/Sync server override preferences -->
+    <string name="toast_override_account_sync_server_done">Mozilla Account/Sync server modified. Quitting the application to apply changes…</string>
```

Comment:
@Delphine I don't think we usually want our Mozilla tradework exposed to be translated. Is this also the case here?

Reply:
We didn't parameterize "Firefox accounts" as per branding guidelines in the past, so we can keep doing the same for "Mozilla account" (again, no capitalization on "account")